src/key_gen/encryptor.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// https://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+use super::Error;
+use crate::id::PublicId;
+use aes::Aes128;
+use block_modes::block_padding::Pkcs7;
+use block_modes::{BlockMode, Cbc};
+use rand::Rng;
+use serde_derive::{Deserialize, Serialize};
+use std::collections::{BTreeMap, BTreeSet};
+
+type Aes128Cbc = Cbc<Aes128, Pkcs7>;
+
+const KEY_SIZE: usize = 16;
+const IV_SIZE: usize = 16;
+
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct Key(pub [u8; KEY_SIZE]);
+
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct Iv(pub [u8; IV_SIZE]);
+
+fn encrypt(data: &[u8], key: &Key, iv: &Iv) -> Result<Vec<u8>, Error> {
+    let cipher =
+        Aes128Cbc::new_var(key.0.as_ref(), iv.0.as_ref()).map_err(|_e| Error::Encryption)?;
+    Ok(cipher.encrypt_vec(data))
+}
+
+// pub fn decrypt(encrypted_data: &[u8], key: &Key, iv: &Iv) -> Result<Vec<u8>, Error> {
+//     let cipher =
+//         Aes128Cbc::new_var(key.0.as_ref(), iv.0.as_ref()).map_err(|_err| Error::Encryption)?;
+//     cipher
+//         .decrypt_vec(encrypted_data)
+//         .map_err(|_err| Error::Encryption)
+// }
```

Comment:
Does this being commented out mean that something is not finished yet?

Reply:
Yes. It may be used for Justification phase.
However, as explained within the code comment, that phase may not be needed.

Reply:
If it turns out to not be needed, do we even need the encryptor at all?

Reply:
Yes, could be.
However, we haven't decided on it yet. So maybe worth to keep it.
So, at least we are reminded there is this feature we may need to provide.

Reply:
OK, got it. Should we maybe raise an issue about this though?

Reply:
Ok. I will

Reply:
raised issue raised issue https://github.com/maidsafe/BLS-DKG/issues/13 as suggested